packages/expo-cli/src/commands/init.ts

```diff
@@ -324,6 +359,46 @@ function logProjectReadyAsync({
   }
 }
 
+async function configureUpdatesProjectFilesAsync(
+  projectRoot: string,
+  initialConfig: BareAppConfig,
+  username: string
+) {
+  const expoPlistPath = path.join(projectRoot, 'ios', initialConfig.name, 'Supporting/Expo.plist');
+  const androidManifestPath = path.join(projectRoot, 'android/app/src/main/AndroidManifest.xml');
+
+  const appUrl = `https://exp.host/@${username}/${initialConfig.name}`;
+  const sdkVersion = (await readConfigJsonAsync(projectRoot)).exp.sdkVersion;
```

Comment:
we should avoid using `readConfigJsonAsync` if possible and use `getConfig` instead. if we decide to switch a template over to app.config.js/ts at some point it would then work right out of the box